src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaRepository.java

```diff
@@ -99,6 +94,30 @@ public void createTopic(final String topic, final int partitionsNum, final int r
         }
     }
 
+    public boolean topicExists(final String topic) throws NakadiException {
```

Comment:
I think just `topic` is better. The names like `topicId` or `topicName` are misleading as for me, because they assume there is something else than id or name (but topic doesn't have anything else)

And I think the Topic class is overcomplication, I don't use it anywhere.